### ISSUE TITLE
Support jira instances with self-signed certificates (on corporate networks)

### ISSUE THREAD

**Issue by Mause** [coddingtonbear/jira-select#2]
That is, support disabling ssl verification

**Comment by coddingtonbear** [comment]
Sounds reasonable to me!  I'm not sure I want to build much UI around that, so what I'm thinking is, maybe, adding a couple command-line flags one can use when running `configure`; maybe two:

* ``--certificate=/path/to/certificate`` -- for if you have access to your self-signed cert
* ``--disable-certificate-verification`` -- for if you really don't mind the lack of verification

Using them would set a setting in your config for that particular instance.

Sound alright to you?

**Comment by coddingtonbear** [comment]
& looks like it'd be super easy to configure the jira instance to use those settings

- https://community.atlassian.com/t5/Jira-questions/JIRA-Python-Connection-Over-HTTPS/qaq-p/376904#M157707

**Comment by Mause** [comment]
Sounds good :) if you can also disable the ssl warnings too, that would also be great.

```py
from urllib3 import disable_warnings
disable_warnings()
```

**Comment by coddingtonbear** [comment]
I don't happen to have a jira version handy that uses a self-signed certificate -- if I were to give you a branch with these changes in them, would you be willing to confirm that they work?

**Comment by Mause** [comment]
Sure!

**Comment by coddingtonbear** [comment]
Ok -- I've pushed up a branch named `2_self_signed_certs` that adds the feature you're hoping for.  Whenever you get a chance, let me know if it works.

**Comment by Mause** [comment]
Looks good - though it confused me a bit at first that the flag had to be before the command:
```sh
jira-shell --disable-certificate-verification configure
```
not
```sh
jira-shell configure --disable-certificate-verification
```

**Comment by coddingtonbear** [comment]
Ahh, yeah!  Sorta unavoidable if we want those options to be available with the normal overrides.  I'll update the "Troubleshooting" note to mention that it's a global configuration flag instead of one just for `configure`.

Just by the way -- in case it's not clear -- you won't have to use that flag after you've run `configure`.

**Comment by coddingtonbear** [comment]
These changes were released as part of 1.6.0.  Thanks for the issue and help verifying the changes @Mause!

Cheers!
